src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -36,7 +36,7 @@
             <argument type="service" id="service_container" />
             <argument>%api_platform.formats%</argument>
             <argument>%api_platform.resource_class_directories%</argument>
-            <argument type="service" id="api_platform.subresource_operation_factory" />
+            <argument type="service" id="api_platform.subresource_operation_factory.cached" />
```

Comment:
Why? `api_platform.subresource_operation_factory` is decorated so you'll have an instance of `CachedSubresourceOperationFactory`, nop?